### ISSUE TITLE
Wrong AppID

### ISSUE THREAD

**Issue by Marty25** [dajver/VKMusicPlayer-Android#1]
Hello! To have an access to vk api and listen music from this site you have to change your appid in here:

"com_vk_sdk_AppId"

You just need to find an official app and use its id. Like this: https://play.google.com/store/apps/details?id=ge.georgeapp.vkmusicx

**Comment by dajver** [comment]
No, you have to go here http://api.xn--41a.ws/ and get api key from there.
